### PR TITLE
GOps: Changes to add Grafana SLO steps to IRM configuration tracker

### PR DESCRIPTION
**Pull request by obetomuniz**
**What is this feature?**

This PR adds SLO to the configuration tracker for Admins. It updates the existing IRM landing page with the a UI structure.

**Who is this feature for?**

For now, will be only for Admin users.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/irm-timetovalue/issues/7

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/types.ts [hunk 1]
```diff
@@ -12,4 +12,5 @@ export const GRAFANA_APP_RECEIVERS_SOURCE_IMAGE: Record<SupportedPlugin, string>
   [SupportedPlugin.Incident]: '',
   [SupportedPlugin.MachineLearning]: '',
   [SupportedPlugin.Labels]: '',
+  [SupportedPlugin.Slo]: '',
```
Comment (soniaAguilarPeiron) [inline public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/types.ts:15]: We don't need this line on this PR. Slo is not going to become a Grafana receiver.

Comment (obetomuniz) [reply public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/types.ts:15]: I see! Initially, I did not add this line, but without it, drone fails. Any recommendation?

**Comment by obetomuniz** [comment]
Closing this PR to avoid conflicts during merge. I'll reopen another one today later.

**Pull request closed without merging**